services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_reports/diagnostics/growthSummary.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react'
+
+import { baseDiagnosticImageSrc, triangleUpIcon, } from './shared'
+
+const barGraphIncreasingIcon = <img alt="Bar chart growth icon" src={`${baseDiagnosticImageSrc}/icons-bar-graph-increasing.svg`} />
+
+const GrowthSummary = ({ showGrowthSummary, skillsGrowth, name, growthSummaryLink, }) => {
```

Comment:
a think a props interface would be nice to know what the data shape is for each prop